share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/head/resources.get.html.ftl

```diff
@@ -90,8 +90,10 @@
       Alfresco.constants.TINY_MCE_SUPPORTED_LOCALES = "${config.global["I18N"].getChildValue("tiny-mce-supported-locales")}";
 
       <#assign aimsConfig = config.scoped["AIMS"] />
-      <#if aimsConfig?? && aimsConfig.enabled.getValue()?boolean == true>
-         Alfresco.constants.AIMS_AUTH_SERVER_URL = "${aimsConfig.authServerUrl.getValue()!""}";
+      <#if aimsConfig?? && (aimsConfig.enabled.getValue() == "true" || aimsConfig.enabled.getValue() == "false")>
```

Comment:
No need for || aimsConfig.enabled.getValue() == "false"
   - we'll just default to false for anything other than "true", including if not set (by the way, does getValue() behave well if not set or null? ).

Reply:
@alexbalmus We're checking for "false" or "true" since we don't want end up having invalid value on the javascript code if someone adds something different that true or false. Another reason for the condition is that FreeMarker's converter from string to boolean (?boolean) crashes if the string value is not "true" or "false" so I couldn;t just convert the value. You'll see that if the value is not "true" or "false", we set the Alfresco.constants.AIMS_ENABLED = false.